### PR TITLE
Nightvision custom keybinds

### PR DESCRIPTION
**Pull request by DillyTheDillster**
Added a section in the config file to allow for custom config keys, allows for special keys such as LeftAlt (default) and normal keys such as letters. Untested at this moment for numpad/number keys and function keys.

### CODE REVIEW

File: MoreShipUpgrades/Misc/PluginConfig.cs [hunk 1]
```diff
@@ -72,6 +75,16 @@ public class PluginConfig
         public float JUMP_FORCE_INCREMENT { get; set; }
         public float DISCOMBOBULATOR_INCREMENT { get; set; }
 
+        public string TOGGLE_NIGHT_VISION_KEY
+        {
+            get => ConfigEntry("Night Vision", "Toggle Night Vision Key", "LeftAlt", "Key to toggle Night Vision");
+            set
+            {
+                // Ensure that the key is valid, if not, fallback to "LeftAlt"
+                var validKeys = new List<string> { "N", "LeftAlt", "RightAlt", "A", "B", "C", /* Add more keys as needed */ };
```
Comment (dNaax) [inline MoreShipUpgrades/Misc/PluginConfig.cs:84]: Are you sure "A" should be a valid key?

Comment (DillyTheDillster) [reply MoreShipUpgrades/Misc/PluginConfig.cs:84]: in testing choosing a key that is already used does not overwrite it, A in this case would not only be a sideways movement, but also trigger night vision if so desired

due to the fact it did not cause any issues, i didnt see a need in disallowing it, this also allows those who use the lethal rebindings mod to use whatever key they desire in the case their movement keys are non defaulted as well


tldr; allowing all keys does not break anything :)
